oh_staff_ui/fixtures/relation-type-data.json

```diff
@@ -0,0 +1,72 @@
+[
+    {
+        "model": "oh_staff_ui.DescriptionType",
```

Comment:
These all use the `DescriptionType` model, but file (and PR) says `RelationType`.  After loading, admin interface shows nothing in the `Relation Types`, but these 10 values in `Description Types`.